lib/convertor.js

```diff
@@ -40,60 +40,136 @@ class MongoToKnex {
         return 'andWhere';
     }
 
-    processField(field, op) {
-        const fieldParts = field.split('.');
+    /**
+     * Determine if statement lives on parent table or if statement refers to a relation.
+     */
+    processStatement(column, op, value) {
+        const columnParts = column.split('.');
```

Comment:
We could improve readability here slightly by using array destructuring e.g.:
```
const [tableName, columnName] = column.split('.');
```
This way we know exactly what's going on :) Couldn't attach a diff here unfortunatelly

Reply:
👍 i have not touched/optimised the existing logic, except of variable names, because it was hard to read.
will update :)

Reply:
hm i don't think this will work if you only have a column string

```const {tableName, columnName} = 'slug'.split('.')```